README.md

```diff
@@ -52,13 +52,23 @@ Start the container
       --ipc=host \
       --user=root \
       --privileged \
-      -v /:/host:Z \
-      -it kubevirt/libvirt:latest
+      --volume=/:/host:Z \
+      --volume=/dev:/host-dev \
+      --volume=/var/run/libvirt:/var/run/libvirt \
+      --volume=/sys:/host-sys \
+      --tty=trye \
+      --detach=true \
+      kubevirt/libvirt:latest
```

Comment:
Using the "latest" tag was never a good idea, and in fact it has just been removed entirely to address https://github.com/kubevirt/libvirt/issues/54.

The section "Versions to use" just above explain how to pick an actual good version (by sha), and uses `kubevirt/libvirt@sha256:1234567890abcdef...` as placeholder. Please use the same placeholder here.